apps/game_backend/lib/game_backend/units/skills/skill.ex

```diff
@@ -4,6 +4,7 @@ defmodule GameBackend.Units.Skills.Skill do
   use GameBackend.Schema
   import Ecto.Changeset
 
+  alias GameBackend.Units.Buffs.Buff
```

Comment:
This alias is wrong right? this module doesn't exists. The right one would be
`GameBackend.Users.Upgrades.Buff`